Assets/Scripts/FlockAgent.cs

```diff
@@ -9,19 +9,24 @@ public class FlockAgent : MonoBehaviour
     public Collider AgentCollider { get; set; }
 
     public List<Transform> Context { get; set; }
-
+    
+    public Kinematic kinematic;
```

Comment:
naming: should be PascalCase if public

Reply:
Oh ok, I didn't know exactly what was the naming convention for c#, but after some research I understand now.

![image](https://user-images.githubusercontent.com/43450580/112916025-b1c36480-90cd-11eb-9cbb-09a3ecfdf953.png)